### ISSUE TITLE
async probe hangs with json file

### ISSUE THREAD

**Issue by danielberndt** [nodeca/probe-image-size#62]
Hello 👋,

I'm a happy user of this library but just observed something very unexpected:

calling `probe(url)` with e.g. [this url](https://uploads-dev-codecks.s3.eu-central-1.amazonaws.com/_tmp/2021/UNtr66GG63/cdx-colors.json) containing a json will result in a never-resolving promise.

It seems to be the case that it hangs for any kind of json file. (even a file containing only `{}`)

I've updated to `7.2.0` and could still observe this behaviour.

**Comment by puzrin** [comment]
Reproduced. Thanks.

```
https://uploads-dev-codecks.s3.eu-central-1.amazonaws.com/_tmp/2021/UNtr66GG63/cdx-colors.json
```

**Comment by rlidwka** [comment]
Fixed, thanks for reporting.

**Comment by danielberndt** [comment]
awesome! Thanks for the super quick fix 🎉

**Comment by puzrin** [comment]
published 7.2.1
